mne/channels/channels.py

```diff
@@ -387,6 +390,10 @@ def set_channel_types(self, mapping):
             else:
                 coil_type = FIFF.FIFFV_COIL_NONE
             self.info['chs'][c_ind]['coil_type'] = coil_type
+        if len(list(unit_changes.keys())) > 0:
```

Comment:
In my opinion this if statement is unnecessary. Leaving it out would do the same. Only ``msg`` would be initialized for nothing if ``unit_changes`` is empty, which is okay, I think.